components/prize_pool/wikis/rainbowsix/prize_pool_custom.lua

```diff
@@ -7,18 +7,48 @@
 --
 
 local Arguments = require('Module:Arguments')
+local Class = require('Module:Class')
 local Lua = require('Module:Lua')
+local String = require('Module:StringUtils')
+local Variables = require('Module:Variables')
 
 local PrizePool = Lua.import('Module:PrizePool', {requireDevIfEnabled = true})
 
+local LpdbInjector = Lua.import('Module:Lpdb/Injector', {requireDevIfEnabled = true})
+local CustomLpdbInjector = Class.new(LpdbInjector)
+
 local CustomPrizePool = {}
 
+local TIER_VALUE = {32, 16, 8, 4, 2}
+local TYPE_MODIFIER = {Online = 0.65}
+
 -- Template entry point
 function CustomPrizePool.run(frame)
 	local args = Arguments.getArgs(frame)
 	local prizePool = PrizePool(args):create()
 
+	prizePool:setLpdbInjector(CustomLpdbInjector())
+
 	return prizePool:build()
 end
 
+function CustomLpdbInjector:adjust(lpdbData, placement, opponent)
+	lpdbData.weight = CustomPrizePool.calculateWeight(
+		lpdbData.prizemoney,
+		Variables.varDefault('tournament_liquipediatier'),
```

Comment:
```suggestion
		tonumber(Variables.varDefault('tournament_liquipediatier')),
```
don't you need a `tonumber` here for the `TIER_VALUE` table to work as intended?

![Screenshot 2022-07-06 12 39 56](https://user-images.githubusercontent.com/75081997/177532423-c33941a2-2da2-4523-8d92-79a8928c8c44.png)

(if changed the `String.isEmpty` check below could also just be a `not` check)